ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -283,4 +301,16 @@ StackLayout {
             }
         }
     }
+
+    TransferOwnershipPopup {
```

Comment:
Same here, why not wrap it in a `Component` instead? We don't have to instantiate this all the time

Reply:
Just a reflection and please share your thoughts! Also @alexandraB99, @alexjba and @micieslak !

I'm a bit divided with the popups instatiations methods we have right now.. I can understand that there are some that are used a lot and it makes sense to have them instantiated once and just use them.. however, I'm more about with the idea of "instantiate-when-use" with the ones that are not as common flows since for me having them in a "common" place, makes me feel we lose the context. 

In that case, if you feel we have 2 instances that can be commonized.. I would move it to, for example "CommunitySettings", and pass the component to the needed sections.. but not sure about moving it to `Popups.qml` just for the reason I wrote above! WDYT?